docs/cron/export.sh

```diff
@@ -55,14 +55,87 @@ split_file sentences.csv
 split_file sentences_CC0.csv
 split_file transcriptions.csv
 
-python3 "$ROOT/docs/cron/split_files.py" \
-    user_languages.csv \
-    links.csv \
-    tags.csv \
-    sentences_in_lists.csv \
-    sentences_with_audio.csv \
-    --indir "$DL_DIR" \
-    --outdir "$TEMP_DIR"
+# split links by language pair
+mysql --skip-column-names --batch tatoeba -e \
+    "SELECT 
+      COALESCE(sentence_lang, '\N'), 
+      COALESCE(translation_lang, '\N'),
+      sentence_id, 
+      translation_id
+     FROM sentences_translations" | \
+  awk -F"\t" -v dir=$TEMP_DIR 'BEGIN {OFS = "\t"} {
+      src_lg = ($1 == "\N" ? "unknown" : $1);
```

Comment:
You need to escape the backslash (`"\\N"`) otherwise AWK will interpret `\N` as `N`, i.e. it ignores the backslash because there is no special escape sequence `\N`.